### ISSUE TITLE
💅 `noDuplicateFontNames` reports false duplicate

### ISSUE THREAD

**Issue by Umbrien** [biomejs/biome#7152]
### Environment information

```bash
CLI:
  Version:                      2.1.4
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.18.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/10.9.3

Biome Configuration:
  Status:                       Not set
  Path:                         unset

Workspace:
  Open Documents:               0
```

### Rule name

lint/suspicious/noDuplicateFontNames

### Playground link

https://biomejs.dev/playground/?code=YwBvAGQAZQAgAHsACgAJAGYAbwBuAHQALQBmAGEAbQBpAGwAeQA6AAoACQAJAHUAaQAtAG0AbwBuAG8AcwBwAGEAYwBlACwACgAJAAkAUwBGAE0AbwBuAG8ALQBSAGUAZwB1AGwAYQByACwACgAJAAkAUwBGACAATQBvAG4AbwAsAAoACQAJAE0AZQBuAGwAbwAsAAoACQAJAEMAbwBuAHMAbwBsAGEAcwAsAAoACQAJAEwAaQBiAGUAcgBhAHQAaQBvAG4AIABNAG8AbgBvACwACgAJAAkAbQBvAG4AbwBzAHAAYQBjAGUAOwAKAH0ACgA%3D&language=css

### Expected result

Shouldn't report duplicate "Mono" font when we have "SF Mono" and "Liberation Mono"

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
As a workaround, you can use quotes to escape the whitespace.

https://biomejs.dev/playground/?code=YwBvAGQAZQAgAHsACgAJAGYAbwBuAHQALQBmAGEAbQBpAGwAeQA6AAoACQAJAHUAaQAtAG0AbwBuAG8AcwBwAGEAYwBlACwACgAJAAkAUwBGAE0AbwBuAG8ALQBSAGUAZwB1AGwAYQByACwACgAJAAkAIgBTAEYAIABNAG8AbgBvACIALAAKAAkACQBNAGUAbgBsAG8ALAAKAAkACQBDAG8AbgBzAG8AbABhAHMALAAKAAkACQAiAEwAaQBiAGUAcgBhAHQAaQBvAG4AIABNAG8AbgBvACIALAAKAAkACQBtAG8AbgBvAHMAcABhAGMAZQA7AAoAfQAKAA%3D%3D&language=css
